reference/config_files/settings.yml.rst

```diff
@@ -173,7 +173,7 @@ The new ``msvc`` compiler is a new, **experimental** one, that is intended to de
 - At the moment it implements a ``compatible_packages`` fallback to Visual Studio compiled packages, that is, previous existing binaries
   compiled with ``settings.compiler="Visual Studio"`` can be used for the ``msvc`` compiler if no binaries exist for it yet.
   This behavior can be opted-out with ``core.package_id:msvc_visual_incompatible`` :ref:`global_conf` configuration.
-- It is not detected by the profile auto-detect, it needs to explicitly be defined in profiles.
+- Starting in conan 1.40.0, ``msvc`` is detected by the profile auto-detect. Before 1.40.0, it needs to be explicitly defined in profiles.
```

Comment:
That's all there is here.

Reply:
But it only happens for VS 2022, isn't it?

```python
 # Visual Studio 2022 onwards, detect as a new compiler "msvc"
    if compiler == "Visual Studio":
        version = Version(version)
        if version == "17":
            compiler = "msvc"
            version = "193"
```
Maybe worth clarifying?

Reply:
That's a good point yes.

Reply:
Thanks!